content/technical/open-source-as-a-service.md

```diff
@@ -0,0 +1,153 @@
+interconnected services, from legal agreements, and so on.
+
+Any non-toy Service is not reproducible.
+
+Launching a clone of the service in order to effectively contribute
+is a high barrier. Certain kinds of contributions clearly are possible.
+But contributors are essentially asked to "fork" the above non-code
+aspects of the service in order to iterate on their changes against
+their real world use cases.
+
+What's more, without true Open Source principles, we are missing a
+way to have a community develop with a center of gravity around
+the service itself. This leads to companies
+[forking the service](https://aws.amazon.com/blogs/opensource/stepping-up-for-a-truly-open-source-elasticsearch)
+in a way that typical open source projects with a true community
+are resilient to.
+
+I believe that if we enable contributions to services, we reap
+the advantages of true Open Source. That means enabling users to make a
+change to a running service and experience that change themselves.
+
+You still with me?
+
+Yes, that means opening a pull request against a running service
+and running that change before it's merged.
+
+This is not as insane as it sounds.
+
+It appears that every technique we need to enable such a capability
+is already in use in modern deployment and operations methodology.
+We just have to connect the dots.
+
+
+# Why should I care?
+
+I've been a part of Open Source for over 20 years now. Although I'm
+just one contributor, together we've changed something fundamental
+about the world. Our lives are too short, and software too complex,
+to have one company or one individual invent everything from scratch.
+So we work together across humanity to accomplish what no
+individual team could. This is our legacy, and it has become
+common place.
+
+And yet there's a very simple principle that drives Open Source:
+
+<div style="font-weight: bold; text-align: center;">
+Open Source thrives when it converts some<br>small percentage of users into contributors.
+</div>
+
+If you interrupt that principle, you starve Open Source. Software as a
+Service does just that: When someone else runs software for you, the
+intuitive mechanisms for users to change that software are unavailable.
+
+"But" you may say, "If the source code for that running service was **open**
+then I could still change it."
+
+Sure you could change the code in some component, but for any reasonably
+complex Service you wouldn't be able to run your changes, experience your
+changes against real world workflows, much less iterate on them until
+they're good enough to share with others.
+
+In reality the threat here is far more fundamental: Users of services
+cannot readily change something in the service not only because the
+processes is operated by someone else ... but because such users of
+a service have explicitly chosen not to be involved in operating
+the software.
+
+If the primary way to use [Postgres was as a Service](https://aws.amazon.com/rds/),
```

Comment:
"as a Service" ?

(i.e.  add quotes to clarify. This took me a few reads to ensure I understood the point)

Reply:
Yes, how about:
"If the primary way how to use Postrgres, would have been to consume it [as a Service](), then that..."

Reply:
Thanks. Good catch. Updated to use quotes. It matches my flavor better ;)